### PR TITLE
Add properties and units

### PR DESCRIPTION
**Pull request by theharq**
Adds the ability to create properties and units

Useful when the user logs in without any property associated.

### CODE REVIEW

File: app/controllers/properties_controller.rb [hunk 1]
```diff
@@ -10,13 +10,43 @@ def show
...
+      redirect_to new_property_unit_path(@property)
+    else
+      render :new
+    end
+  end
+
   private
 
   def redirect_to_desired_path
+    redirect_to new_property_path if has_no_properties?
```
Comment (tadp) [inline app/controllers/properties_controller.rb]: Can we have it always redirect to dashboard?

**Comment by tadp** [comment]
I think this is great to be able to add more properties.

WDYT of adding a button that redirects to add new properties path on the dashboard? Because people with more than one property would want to keep adding properties after they first login even if they only initially signed up with one property. (Also, people buy and sell properties all the time)

**Comment by theharq** [comment]
Updated

**Comment by tadp** [comment]
tested out and it works 👍

**Pull request merged**